packages/blocks/embed/package.json

```diff
@@ -16,7 +16,7 @@
     "build": "yarn clean; cross-env NODE_ENV=production webpack --mode production && yarn schema",
     "build:dev": "yarn clean; cross-env NODE_ENV=development webpack --mode development",
     "clean": "rimraf ./dist/",
-    "dev": "cross-env NODE_ENV=development webpack-dev-server -d --port 3000 --config webpack-dev-server.config.js --open",
+    "dev": "cross-env NODE_ENV=development webpack-dev-server -d --port 9000 --config webpack-dev-server.config.js --open",
```

Comment:
Makes it possible to test blocks with a frontend dev server running simultaneously.